spec/controllers/application_controller/ci_processing_spec.rb

```diff
@@ -120,9 +120,14 @@
                                         :controller             => "ems_cloud"
                                       )
       allow(controller).to receive(:drop_breadcrumb)
+      cloud_manager_stub = double('CloudManager',
+                                  :supports_discovery? => true,
+                                  :ems_type            => 'example',
+                                  :description         => 'Example Manager')
+      allow(ManageIQ::Providers::CloudManager).to receive(:subclasses).and_return([cloud_manager_stub])
```

Comment:
@chrisarcand jfyi, this is as good as I can get. See also https://github.com/rails/rails/issues/25141